packages/ui/src/utils/button.ts

```diff
@@ -31,10 +31,15 @@ const focusOutline = css<{
    * disabled button, the overlay of the disabled button would be above the
    * outline, making the outline appear to be partly cut off.
    */
-  &:focus::after {
+  &:focus-within {
+    outline: none;
+  }
+
+  &:focus-within::after {
```

Comment:
Radix UI's `<Tabs />` uses `focus-within` rather than `focus`, so this addresses that change.